lib/wickedbase.pm

```diff
@@ -148,6 +150,23 @@ sub get_current_ip {
     return;
 }
 
+=head2 download_all
+
+Download all files from data/wicked into WICKED_DATA_DIR.
+This method is used by before_test.pm.
+=cut
+sub download_all {
```

Comment:
`all` is not very clear for me . what about `download_data_dir` ?

Reply:
ok